modules/update-service-overview.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='update-service-overview-{context}']
+== The {product-title} update service
+
+The {product-title} update service is the hosted service that provides over the air updates to both 
+{product-title} and RHCOS. It provides a graph, or diagram that contain
+_vertices_ and the _edges_ that connect them, of component Operators. The edges
+in the graph show which versions you can safely upgrade to. The cluster version
+Operator checks with the {product-title} update service and determines valid upgrades and upgrade paths
+based on current component versions and information in the graph. If you
+configure it to do so, the {product-title} update service sends the release artifacts that it needs to
+perform the upgrade to your image registry, and the cluster version Operator
+upgrades your cluster. By accepting automatic updates, you can automatically
+keep your cluster up to date with the most recent compatible components.
+
+To allow the {product-title} update service to provide only compatible updates, a release verification
+pipeline exists to drive automation. Each release artifact is verified for
+compatibility with supported cloud platforms and system architectures as well
+as other component packages. After the pipeline confirms the suitability of a 
```

Comment:
You cover testing update payloads in isolation (before we push a node to the update graph), but it would be good to have some words around testing the upgrades (before we push an edge).  CC @steveeJ who actually understands how graph updates work, because I could be getting this wrong ;).

Reply:
What's the average reader interested in, i.e. who's the average reader for this document? Might they be interested in [OpenShift 4 Channels][1]? The *suitability* property is determined by the [Release Controller][2].

[1]: https://docs.google.com/document/d/1AeuliIIu2VN-sF8ECpFMygrDoz4D2cZ8B44sZFBw4P8/edit?pli=1#heading=h.xx9vu52n421x

[2]: https://docs.google.com/document/d/1PAHSF86Un6CdG7STX-vIGbrgjVTnVpDOSBplos0U8Rk/edit#heading=h.5k1h50oq8lyi

Reply:
I made a separate issue to document a user story about understanding and configuring updates, and I've added a note to update this section. Thanks!